docs/source/_templates/home.html

```diff
@@ -437,31 +438,45 @@ <h3 class="h-card__title">Mathematics</h3>
                         </div>
                     </li>
                     <li class="physics-simulation">
-                        <div class="grid-container">
-                            <div class="visualization-images">
-                                <div class="image-grid">
-                                    <div>
-                                        <a href="fury-pybullet.html"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/physics_domino.gif" alt="physics domino" align="middle"></a>
-                                    </div>
-                                    <div>
-                                        <a href="fury-pybullet.html"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/physics_bricks_multi_actor.gif" alt="physics bricks" align="middle"></a>
-                                    </div>
-                                    <div>
-                                        <a href="fury-pybullet.html"><img src="https://raw.githubusercontent.com/fury-gl/fury-communication-assets/main/physics_collision.gif" alt="physic collision" align="middle"></a>
-                                    </div>
-                                    <div>
-                                        <a href="auto_examples/index.html#demos"><img src="https://github.com/fury-gl/fury-communication-assets/raw/main/brownian_motion.gif" alt="brownian motion" align="middle"></a>
-                                    </div>
-                                    <div>
-                                        <a href="auto_examples/index.html#demos"><img src="https://github.com/fury-gl/fury-communication-assets/raw/main/em_wave.gif" alt="Em wave" align="middle"></a>
-                                    </div>
-                                    <div>
-                                        <a href="auto_examples/index.html#demos"><img src="https://github.com/fury-gl/fury-communication-assets/raw/main/helical_path.gif" alt="Helical path" align="middle"></a>
-                                    </div>
+                        <h6 class="tabs-info">With FURY, it becomes simple to produce multiple physics animations with ease.</h6>
```

Comment:
This text is slightly too big in mobile version. Can you update this text during mobile version.

Also, it would be great to use the same font family as the other section (scientific domains, etc...)